hvantk/datasets/cptac_phospho_datasets.py

```
@@ -0,0 +1,364 @@
+
+        # Fetch normal phospho data (may be empty for some cancer types)
+        normal_sites = []
+        try:
+            normal_df = ds.get_phosphoproteomics(source="umich", tissue_type="normal")
+            if len(normal_df) > 0:
+                logger.info("Normal: %d samples x %d columns", *normal_df.shape)
+                normal_sites = extract_phospho_sites(normal_df, ct, tissue_type="normal")
+                write_intermediate_tsv(normal_sites, normal_tsv)
+            else:
+                logger.info("No normal samples available for %s", ct)
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)